ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -351,7 +353,11 @@ export default class ConfirmPageContainer extends Component {
             <PageContainerFooter
               onCancel={onCancel}
               cancelText={t('reject')}
-              onSubmit={isSetApproveForAll ? onSetApprovalForAll : onSubmit}
+              onSubmit={
+                isSetApproveForAll && isApprovalOrRejection
+                  ? onSetApprovalForAll
+                  : onSubmit
```

Comment:
The PR works as intended based on https://github.com/MetaMask/metamask-extension/pull/16898#issuecomment-1346441605. That is, the revoke warning is no longer shown. 

I think we need to update the description

Reply:
The description? You mean the explanation of the pull request?   @digiwand

Reply:
Hi @EresDev,

yeah. It seems to me
> I have updated the warning message for revoking approval for all NFTs along with the action button (Approve, Reject) titles to clarify the intent.

is now outdated. It seems we have removed the revoke approval warning altogether if I'm not mistaken

Reply:
Ok @digiwand. I have updated the explanation description in https://github.com/MetaMask/metamask-extension/pull/16898. I think @danjm can use that to update the explanation of this PR as well.